writer.go

```diff
@@ -40,7 +48,16 @@ type Writer struct {
 	lineCount int
 }
 
-// New returns a new Writer with defaults
+func InitializeTerminalWidth() {
```

Comment:
Doesn't seem like this needs to be exported. Will be great if you could please move it to writer's initializer.